src/lib/provable/gadgets/elliptic-curve.ts

```diff
@@ -65,6 +65,11 @@ function add(p1: Point, p2: Point, Curve: { modulus: bigint }) {
     return Point.from(p3);
   }
 
+  assert(
+    Curve.modulus > l2Mask + 1n,
+    'Base field moduli smaller than 2^176 are not supported'
+  );
```

Comment:
this is needed because we can use the addition gadget directly, without the `ForeignCurve` wrapper